Description: output to standard output while building.
 The upstream build system puts all output in a log file rather than to standard
 output. This can cause build timeouts on slow architectures and if the build
 truely hangs makes it difficult to determine where it hung. Modify the build
 system to output to standard output as it goes along.
 
Author: Peter Michael Green <plugwash@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- nacl-20110221.orig/do
+++ nacl-20110221/do
@@ -38,7 +38,7 @@ mkdir -p "$bin"
 mkdir -p "$lib"
 mkdir -p "$include"
 
-exec >"$top/log"
+exec | tee "$top/log"
 exec 2>&1
 exec 5>"$top/data"
 exec </dev/null
